### PR TITLE
Improvement/upstream updates

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request adds types for all variables.  I also pull in upstream updates while I'm at it.

## 💭 Motivation and Context

I am going through all the Terraform module repositories to remove non-`alias` variables from proxy providers.  This repository did not require that change, but I noticed that the variable types were not always specified.

## 🧪 Testing

All pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/lineage.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```
Comment (dav3r) [inline .github/lineage.yml]: This should be `skeleton-tf-module`, not `skeleton-generic`.

**Review by dav3r — APPROVED** [review]
Good to go once you fix the incorrect skeleton in `.github/lineage.yml`. 🚀

File: .github/lineage.yml (hunk 1 above)
Comment (jsf9k) [reply .github/lineage.yml]: See commit 827372d.

**Review by mcdonnnj — APPROVED** [review]
I'm not sure what happened to my previous review. You took care of the one thing I mentioned so LGTM ✔

**Pull request merged**